locales/en-US/app.ftl

```diff
@@ -182,6 +182,9 @@ newsletterFooterBody = Find out about new experiments and see test results for e
 newsletterFooterSuccessHeader = Thanks!
 newsletterFooterSuccessBody = If you haven't previously confirmed a subscription to a Mozilla-related newsletter you may have to do so. Please check your inbox or your spam filter for an email from us.
 
+localeWarningTitle = This experiment is only available in English.
```

Comment:
@lmorchard @chuckharmston yeah, my guess is that for the foreseeable future experiments (other than the upcoming German experiment) will start in English and be localized later. Should that cease to be the case, we'll need to revisit this string.